tests/migrations/test_writer.py

```diff
@@ -152,6 +152,20 @@ def test_args_kwargs_signature(self):
             "),",
         )
 
+    def test_keyword_only_args_signature(self):
+        operation = custom_migration_operations.operations.KeywordOnlyArgsOperation(
+            kwarg1=1, kwarg2=2
+        )
+        buff, imports = OperationWriter(operation, indentation=0).serialize()
+        self.assertEqual(imports, {"import custom_migration_operations.operations"})
+        self.assertEqual(
+            buff,
+            "custom_migration_operations.operations.KeywordOnlyArgsOperation(\n"
+            "    kwarg1=1,\n"
+            "    kwarg2=2,\n"
+            "),",
+        )
```

Comment:
Can we instead add a test for `CreateCollation` in `tests.postgres_tests.test_operations.CreateCollationTests`?